src/dragonball/src/error.rs

```diff
@@ -0,0 +1,41 @@
+// Copyright (C) 2022 Alibaba Cloud. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+//
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Portions Copyright 2017 The Chromium OS Authors. All rights reserved.
```

Comment:
Please can you provide details in the commit(s) showing where this code originated (project, URL, version / release, date). For example, was the Amazon copyright material taken from Firecracker, or some other project?

Same comment for the other files with 3rd party copyright statements.

Reply:
`Acknowledgement` part added in the README in dragonball helps explain this :)